src/lib/pushes/model/PushPayload.ts

```diff
@@ -0,0 +1,60 @@
+import { DeviceType } from '../../templates/model/TemplatePushPayload'
+
+export class PushPayload {
+  audience: any
```

Comment:
Should we type `audience` here, then you can reference that as the type on the backend where you're building the payload?